pickup/lib/car.js

```diff
@@ -50,5 +50,5 @@ export async function checkCar (car) {
     }
   }))
   const carCid = createCarCid(sha256)
-  return { carCid, carSize, report }
+  return { carCid: carCid.toString(), carSize, report }
```

Comment:
we're working with cid strings everywhere else in this code, as we're handed it from and sqs message. We log the carCid to loki via pino, and it was serialising the cid object instead of calling toString on it, so we normalise it here.